hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionServerWrapperImpl.java

```diff
@@ -433,8 +433,9 @@ public List<String> getWALExcludeDNs() {
     if (excludeDatanodeManager == null) {
       return Collections.emptyList();
     }
-    return excludeDatanodeManager.getExcludeDNs().entrySet().stream()
-      .map(e -> e.getKey().toString() + ", " + e.getValue()).collect(Collectors.toList());
+    return excludeDatanodeManager.getExcludeDNs().entrySet().stream().map(e -> e.getKey().toString()
+      + " - " + e.getValue().getSecond() + " - " + e.getValue().getFirst())
```

Comment:
In the old code we use ',' as separator, but in the new code we change to all use '-'? Could you please give some examples about how it looks like in the new implementation?